recipes/pypeln/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "pypeln" %}
+{% set version = "0.4.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "c9a3b72f981441e0ea7576cf54c4cd77289b6a2322497da43581cf9b47999ff4"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - poetry
+    - stopit >=1.1.2,<2.0.0
+  run:
+    - python
+    - typing-extensions
+    - dataclasses
+    - stopit >=1.1.2,<2.0.0
+
+test:
+  imports:
+    - pypeln
+    - pypeln.process
+    - pypeln.process.api
+    - pypeln.sync
+    - pypeln.sync.api
+    - pypeln.task
+    - pypeln.task.api
+    - pypeln.thread
+    - pypeln.thread.api
```

`pip check` is always helpful
```suggestion
    - pypeln.thread.api
  requires:
    - pip
   commands:
    - pip check
```